ignite/services/network/join.go

```diff
@@ -61,21 +61,14 @@ func (n Network) Join(
 	}
 
 	// get the peer address
-	if o.publicAddress != "" {
```

Comment:
I think this check was related to make passing public addresses optional, maybe we should keep it?

And maybe check if we are on Gitpod and if not bypass the `o.publicAddress != ""` then?

Reply:
The PR is not ready. I pushed a draft to make experimentations on Gitpod and this is not the issue